examples/bootstrap-script/pylintrc

```diff
@@ -0,0 +1,378 @@
+[MASTER]
```

Comment:
Was this file committed by accident? I don't think we should have a full pylintrc in one example directory.

Reply:
Instead of adding this large config file that's not part of the example, a better fix is probably to add this line to the top:

```python
c = get_config()  # noqa: F821
```

This ensures the variable is defined and the linter satisfied for flake8 (the linter used in this repo)

Reply:
@minrk Have changed. Thanks